src/builder/server.js

```diff
@@ -107,6 +107,12 @@ export default (config, options = {}) => {
     noInfo: true,
   }))
   server.use(options.baseUrl.pathname, router)
+  // Serve staticAssets on development
+  if (options.staticAssets !== undefined) {
+    const staticAssets = options.staticAssets
+    const route = options.baseUrl.pathname + staticAssets.route
+    server.use(route, express.static(staticAssets.path))
+  }
```

Comment:
Cool, otherwise we can have issue if a static file as the same name as a webpack outputed file (you will have the static one, and won't understand why you don't get the webpack version).